.github/workflows/release.yml

```diff
@@ -54,6 +60,16 @@ jobs:
           sudo apt-get install -y make g++ libssl-dev gcc-aarch64-linux-gnu
           rustup target add aarch64-unknown-linux-gnu
 
+      - name: Install Rust crosscompile tools (musl)
+        if: ${{ contains(matrix.target, 'musl') }}
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y musl-tools
+          if [[ "${{ matrix.target }}" == "aarch64-unknown-linux-musl" ]]; then
+            sudo apt-get install -y gcc-aarch64-linux-gnu
```

Comment:
do we also need to do the same thing for [gcc-x86-64-linux-gnu](https://packages.debian.org/sid/gcc-x86-64-linux-gnu)

Reply:
I don't think so